server/keyshare/tasks/tasks.go

```diff
@@ -66,6 +75,19 @@ func runWithTimeout(fn func(ctx context.Context)) error {
 	return ctx.Err()
 }
 
+func hasMailRevalidation(conf *Configuration, db *keyshare.DB) bool {
+	ctx, cancel := context.WithTimeout(context.Background(), taskTimeout)
+	defer cancel()
+
+	c, err := db.ExecCountContext(ctx, "SELECT true FROM information_schema.columns where table_schema='irma' AND table_name='emails' AND column_name='revalidate_on'")
+	if err != nil {
+		conf.Logger.WithField("error", err).Error("could not query the schema for column emails.revalidate_on, therefore revalidation is disabled")
```

Comment:
If this error occurs it is because querying itself actually failed. This could happen due to a db server issue, a rights issue on the information_schema.columns table etc. Normally this won't happen: if the `revalidate_on` column does not exist, c == 0 and the function just returns false. 

But if the query does throw an error, I could be a severe issue, so we'd better log it as an error. Because it might be an isolated problem we do not halt but just return false, allowing further processing of the tasks. 

To prevent duplicate logging I've moved the warning log into `hasMailRevalidation` so it now either returns true, or false (with an error or a warning log entry).